Accordion.js

```diff
@@ -75,19 +72,31 @@ class Accordion extends Component {
 
     return (
       <View {...viewProps}>
-      {this.props.sections.map((section, key) => (
-        <View key={key}>
-          <TouchableHighlight onPress={() => this._toggleSection(key)} underlayColor={this.props.underlayColor}>
-            {this.props.renderHeader(section, key, this.state.activeSection === key)}
-          </TouchableHighlight>
-          <Collapsible collapsed={this.state.activeSection !== key} {...collapsibleProps}>
-            {this.props.renderContent(section, key, this.state.activeSection === key)}
-          </Collapsible>
-        </View>
-      ))}
+        {this.props.sections.map((section, key) =>
+          <View key={key}>
+            <TouchableHighlight
+              onPress={() => this._toggleSection(key)}
+              underlayColor={this.props.underlayColor}
+            >
+              {this.props.renderHeader(
+                section,
+                key,
+                this.state.activeSection === key
+              )}
+            </TouchableHighlight>
+            <Collapsible
+              collapsed={this.state.activeSection !== key}
+              {...collapsibleProps}
+            >
+              {this.props.renderContent(
+                section,
+                key,
+                this.state.activeSection === key
+              )}
+            </Collapsible>
+          </View>
+        )}
       </View>
     );
   }
 }
-
-module.exports = Accordion;
```

Comment:
Ohh? I thought it wouldn't be a problem since react native ships with babel. https://facebook.github.io/react-native/docs/javascript-environment.html#javascript-syntax-transformers